api/src/test/java/uk/gov/dhsc/htbhf/hmrc/HMRCIntegrationTests.java

```diff
@@ -0,0 +1,139 @@
+        //Then
+        assertResponseCorrectWithHouseholdDetails(response, HOUSEHOLD_INDENTIFIER, ELIGIBLE);
+        verify(restTemplateWithIdHeaders).postForEntity(HMRC_URL, aValidHMRCEligibilityRequest(), EligibilityResponse.class);
+    }
+
+    @ParameterizedTest(name = "Should return eligible response for claimant [{0}] Simpson stored in UC household table")
+    @CsvSource({"Homer, EB654321B",
+            "Marge, EB876543A"})
+    void shouldReturnEligibleWhenMatchesHouseholdInDatabase(String parentName, String nino) {
+        //Given
+        householdRepository.save(aHouseholdWithChildrenAged6and24months());
+        PersonDTO person = aValidPersonBuilder().firstName(parentName).nino(nino).build();
+        EligibilityRequest eligibilityRequest = anEligibilityRequestWithPerson(person);
+
+        //When
+        ResponseEntity<EligibilityResponse> response = callService(eligibilityRequest);
+
+        //Then
+        assertResponseCorrectWithHouseholdDetails(response, HOUSEHOLD_INDENTIFIER, ELIGIBLE);
+        verifyZeroInteractions(restTemplateWithIdHeaders);
+        householdRepository.deleteAll();
+    }
+
+    @Test
+    void shouldReturnNoMatchWhenMatchesNinoButNotNameInDatabase() {
+        //Given
+        householdRepository.save(aHousehold());
+        PersonDTO person = aValidPersonBuilder().lastName("noMatch").nino(HOMER_NINO).build();
+        EligibilityRequest eligibilityRequest = anEligibilityRequestWithPerson(person);
+
+        //When
+        ResponseEntity<EligibilityResponse> response = callService(eligibilityRequest);
+
+        //Then
+        assertResponseCorrectWithStatusOnly(response, NOMATCH);
+        verifyZeroInteractions(restTemplateWithIdHeaders);
+        householdRepository.deleteAll();
+    }
+
+    private ResponseEntity<EligibilityResponse> callService(EligibilityRequest eligibilityRequest) {
+        return restTemplate.exchange(buildRequestEntity(eligibilityRequest), EligibilityResponse.class);
+    }
+
+    private void assertResponseCorrectWithHouseholdDetails(ResponseEntity<EligibilityResponse> response, String householdIdentifier, EligibilityStatus status) {
+        EligibilityResponse expectedResponse = aValidEligibilityResponseBuilder()
+                .householdIdentifier(householdIdentifier)
+                .eligibilityStatus(status)
+                .build();
+        assertResponseCorrect(response, expectedResponse);
+    }
+
+    private void assertResponseCorrectWithStatusOnly(ResponseEntity<EligibilityResponse> response, EligibilityStatus status) {
+        EligibilityResponse expectedResponse = EligibilityResponse.builder()
+                .eligibilityStatus(status)
+                .build();
+        assertResponseCorrect(response, expectedResponse);
+    }
+
+    private void assertResponseCorrect(ResponseEntity<EligibilityResponse> response, EligibilityResponse expectedResponse) {
+        AssertionsForInterfaceTypes.assertThat(response.getStatusCode()).isEqualTo(OK);
```

Comment:
That's the wrong static import, plus for below too.

Reply:
Good spot, hopefully correct now.